### PR TITLE
BUG: Call draw before adding attribution (#95)

### LINKED ISSUE #95: BUG: wrapping depends on how ax is created
So I found the use case where wrapping does not work fine:

With

```
import geopandas
from pysal.lib.examples import get_path
texas = geopandas.read_file(get_path('us48.shp')).set_index('STATE_ABBR').loc[['TX'], 'geometry']
texas.crs = {'init': 'epsg:4326'}
texas = texas.to_crs(epsg=3857)
```

then letting the `ax` be created by geopandas, it works fine:

```
ax = tx.plot(alpha=0.5)
ctx.add_basemap(ax)
```

image

but manually creating the axes not (the extent comes from the figure above):

```
extent = (-11945319, -10336026, 2910477, 4438236)
fig, ax = plt.subplots()
ax.set_aspect('equal')
ax.axis(extent)
ctx.add_basemap(ax)
```

gives

image

No idea at the moment what could be the difference (setting the aspect to equal also did not help).

### PR DESCRIPTION
**Pull request by darribas**
Closes https://github.com/darribas/contextily/issues/95

### CODE REVIEW

File: contextily/plotting.py [hunk 1]
```diff
@@ -179,6 +180,7 @@ def add_attribution(ax, att=ATTRIBUTION, font_size=ATTRIBUTION_SIZE):
                           Matplotlib axis with `x_lim` and `y_lim` set in Web
                           Mercator (EPSG=3857) and attribution text added
     """
+    draw()
```
Comment (jorisvandenbossche) [inline contextily/plotting.py]: can you add a comment here?

Comment (darribas) [reply contextily/plotting.py]: A comment as in explaining where that comes from?

Comment (darribas) [reply contextily/plotting.py]: Done

Comment (jorisvandenbossche) [reply contextily/plotting.py]: Thanks!

**Comment by jorisvandenbossche** [comment]
CI was green, so merged it!

**Pull request merged**